### DISCUSSION TITLE
installation on amrv8

### DISCUSSION THREAD

**Discussion by surihost** [dani-garcia/vaultwarden#1742] [Q&A]
Hi,

How to install vaultwarden on armv8 ?

**Comment by mqus** [comment]
Your question is severely lacking context and no one can answer it like that ;)

armv8 is only the architecture(what language the processor speaks). On Which OS (Linux Distribution?) and how (docker/build from source/package?) do you want to install it? Which database backend would you want (depends on how powerful your machine is and how much load it needs to serve)?
